eng/pipelines/common/global-build-job.yml

```diff
@@ -183,6 +205,6 @@ jobs:
       inputs:
         PathtoPublish: '$(Build.SourcesDirectory)/artifacts/log/'
         PublishLocation: Container
-        ArtifactName: Logs_Build_${{ parameters.osGroup }}_${{ parameters.osSubGroup }}_${{ parameters.archType }}_${{ parameters.buildConfig }}_${{ parameters.nameSuffix }}
+        ArtifactName: ${{ parameters.publishLogsArtifactPrefix }}Logs_Build_${{ parameters.osGroup }}_${{ parameters.osSubGroup }}_${{ parameters.archType }}_${{ parameters.buildConfig }}_${{ parameters.nameSuffix }}
```

Comment:
We should use GCC as the name suffix if possible and avoid adding all this prefix tracking.

Reply:
This was copied from corclr script.

Reply:
I understand that, but I think if we’re using this template, we should follow the patterns of this template even if they differ from the CoreCLR template.